packages/api/src/api/helix/moderation/HelixBlockedTerm.ts

```diff
@@ -0,0 +1,67 @@
+import { DataObject, rawDataSymbol, rtfm } from '@twurple/common';
+
+/** @private */
+export interface HelixBlockedTermData {
+	broadcaster_id: string;
+	created_at: string;
+	expires_at: string;
+	id: string;
+	moderator_id: string;
+	text: string;
+	updated_at: string;
+}
+
+/**
+ * Information about a word or phrase blocked in a broadcaster's channel.
+ */
+@rtfm<HelixBlockedTerm>('api', 'HelixBlockedTerm', 'id')
+export class HelixBlockedTerm extends DataObject<HelixBlockedTermData> {
+	/**
+	 * The ID of the broadcaster that owns the list of blocked terms.
+	 */
+	get broadcasterId(): string {
+		return this[rawDataSymbol].broadcaster_id;
+	}
+
+	/**
+	 * The UTC date and time (in RFC3339 format) of when the term was blocked.
+	 */
+	get createdAt(): string {
```

Comment:
For all date properties, please create `Date` instances and name them like `creationDate`. (I missed this in the last PR too, please change that as well)

Reply:
Fixed.